src/main/java/com/sda/auction/entities/Category.java

```diff
@@ -0,0 +1,29 @@
+package com.sda.auction.entities;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import javax.persistence.*;
+import java.util.List;
+
+@Entity
+@Getter
+@Setter
```

Comment:
Instead of these, we can use @Data
@Data will add ToString 
Be aware of Infint Loop Problem